packages/replay/src/types.ts

```diff
@@ -182,32 +182,39 @@ export interface WorkerResponse {
 
 export type AddEventResult = void;
 
-export interface SampleRates {
+interface SessionAndPluginOptions {
```

Comment:
l: Hmm imho this interface does not really help anymore now. I'd rather just put the options directly on both the SessionOptions and the ReplayPluginOptions? Or only put them on the SessionOptions and partially pick them into the ReplayPluginOptions (or vice versa) like:

```ts
interface SessionOptions {
  sessionSampleRate: number;
  allowBuffering: boolean;
  stickySession: boolean;
}

interface ReplayPluginOptions extends Pick<SessionOptions, 'sessionSampleRate' | 'stickySession'> {
}
```

?